packages/x-date-pickers-pro/src/internals/hooks/useDesktopRangePicker/useDesktopRangePicker.tsx

```diff
@@ -127,7 +125,8 @@ export const useDesktopRangePicker = <
         return;
       }
 
-      actions.onDismiss();
+      // This direct access to `providerProps` will go away once the range fields stop having their views in a tooltip.
+      providerProps.privateContextValue.dismissViews();
```

Comment:
We could just go with 'dismiss()', WDYT?

Reply:
Indeed, the `views` is not needed, we don't say `setViewOpen` but just `setOpen`.

WDYT about being extra explicit here and name if `acceptValueAndClose`?
I had to check the code a few time to understand if I should use `onDismiss` or `onCancel`

Reply:
Wait, the difference between `acceptValueChanges` and `dimiss` is even smaller :laughing: 
The only case where `acceptValueChanges` fires `onAccept` and dismiss does not is if you provide a default value to the picker, open it and closes it without any change. `acceptValueChanges` will call `onAccept` with the default value, `dismiss` won't...

I think we could even use `acceptValueChanges` everywhere here, this looks like an inconsistency.

Reply:
I updated the migration guide but did not touh the name or the behavior yet.